src/ert/scheduler/scheduler.py

```diff
@@ -83,8 +76,31 @@ def kill_all_jobs(self) -> None:
         for task in self._tasks.values():
             task.cancel()
 
-    def stop_long_running_jobs(self, minimum_required_realizations: int) -> None:
-        pass
+    async def _stop_long_running_jobs(self, minimum_required_realizations: int) -> None:
+        LONG_RUNNING_FACTOR = 1.25
```

Comment:
Style: `UPPER_SNAKE_CASE` variable defined inside a function. Should be moved to class or module namespace.

Also, what does this control and why 1.25? Add comment?

Reply:
Right, I gave it up for discussion since in "jobqueue" it is this weird constant (1.25), which is essentially never changed. Neverthelles I would either have as 1) just a number or 2) if this is to change at some point it should be a function parameter (defaulted to 1.25).

Reply:
FYI, I opted for nr 2.